tests/setupSpec14.js

```
@@ -0,0 +1,262 @@
+    gt: jest.fn().mockReturnThis(),
+    lt: jest.fn().mockReturnThis(),
+    gte: jest.fn().mockReturnThis(),
+    lte: jest.fn().mockReturnThis(),
+    order: jest.fn().mockReturnThis(),
+    limit: jest.fn().mockReturnThis(),
+    single: jest.fn().mockResolvedValue({
+      data: { id: 'mock-id', created_at: new Date().toISOString() },
+      error: null
+    }),
+    // Default mock responses for common queries
+    then: jest.fn().mockResolvedValue({
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)